src/Options/Options.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+import { LIKER_LAND_URL } from '../constant';
 
-const Options = () => {
-    return <div>Hello World</div>;
-};
+
+const civicLikerURL = `${LIKER_LAND_URL}/civic?utm_soure=extension`;
+const logoutURL = `${LIKER_LAND_URL}/logout`;
+
+const background = browser.extension.getBackgroundPage();
+
+class Options extends React.Component {
+  handleLogout(e: React.MouseEvent) {
```

Comment:
Seems not need to be instance method?

Reply:
Does react normally put click handlers as instance method, or its not relevant?

Reply:
Don't know the current convention (functional component?), but when using instance method as handlers, you need some workaround to bind `this` (e.g. `this.clickHandler = this.clickHandler.bind(this)` in constructor)

Reply:
if the function is no need to use `this`, then it is fine. Otherwise, for convenient, we can use an arrow function to skip the `.bind(this)` in the constructor

Reply:
But I heard that arrow function creates a new function object for every execution, which may have performance issue?

Reply:
Though performance should not be an issue in our case